### ISSUE TITLE
Allow channel change

### ISSUE THREAD

**Issue by cyuste** [pdcarroll/react-kinesis-webrtc#19]
Hello @pdcarroll ,

I've created a [sandbox](https://codesandbox.io/s/suspicious-resonance-t7inu3) with all the required modules to reproduce the issue:

Amplify - not really needed, but useful for authentication
A couple of webrtc streams available, test1 and test2. Both use the same test video, but you can easily see when the feed is changed
A react app with `react-kinesis-webrtc` package installed from npm and as a component, with my suggested changes

# Desired behaviour
- Create an account
- Login. Video feed takes around 20s to load, so be patient!
- Try to use the other video from the dropdown
- After a while, you see the other feed

# To reproduce the issue
- Go to `components/video-player/video-frame/index.tsx` comment line 4 and uncomment line 7

# Warning!
**Codepen** is not working 100% of the times, sometimes has the expected behaviour sometimes doesn't load the video. If so, please download the [repo](https://github.com/cyuste/test-react-video-ts) and run locally.
As a side note, `npm i --legacy-peers-deps` is required because the library is only for react@17 (but works just fine in v18)

**Comment by pdcarroll** [comment]
@cyuste Sorry for the lapse, I haven't had time to look at this recently. Thanks for putting this test repo together; I've got it running locally and am able to repro the error per your instructions. I'm going to debug some more, will keep you posted.

**Comment by pdcarroll** [comment]
@cyuste I created a PR on your test repo. I reverted some of the changes you made, as [this line](https://github.com/cyuste/test-react-video-ts/blob/master/src/components/react-kinesis-webrtc/hooks/useViewer.ts#L229) that you added seemed to make channel switching work with no other changes required (that's what I observed, but I also could be missing something).

Let me know if that works out for you.

**Comment by cyuste** [comment]
Hello @pdcarroll,
Sure, let me run a few tests in the actual application and see how it works.
Thank you very much for your help and my apologies for the spanish comments (I saw them reviewing your PR)

**Comment by pdcarroll** [comment]
@cyuste Thanks! When I run your test app, I don't see any video feed at all, I'm just relying on the viewer debug logs in the console (and the error that occurs when switching channels). So it's possible I'm not reproducing the issue exactly. 

No worries about the comments, I should know more Spanish, but I'm an ignorant American 😝

**Comment by cyuste** [comment]
@pdcarroll, yes I shut down the instance a few days ago... I've just turned it on again and feeds are working

**Comment by pdcarroll** [comment]
@cyuste Got it, I was able to test properly with the video feed when I tried just now (I limited the amount of time I was using the channels for your sake). It appears that the changes in my branch are still working with the video feed.

**Comment by cyuste** [comment]
@pdcarroll , yes I ran a few tests and I agree, with your version is enough to allow channel change. If you could include it in your package that would be great.
One last question, regarding types: They are not being published in the package, are they? I was playing a little bit with tsconfig and rollup but I'm afraid I've never used rollup before so if you could please include them that would be great.

Thank you very much for all your help
